maven/bnd-reporter-maven-plugin/README.md

```diff
@@ -0,0 +1,212 @@
+                <templateType>xslt</templateType>
+                <parameters>
+                    <myParam1>myValue</myParam1>
+                    <myParam2>myValue</myParam2>
+                <parameters>
+            </report>
+        </reports>
+    </configuration>
+```
+> Note: If a file with the same name as the exported report but with a template file extension is found in the same folder as the exported report, this file will be used to transform the report instead of the optionally defined template attribute. This allows to quickly customize a report without redefining an inherited configuration.
+
+#### Internationalisation
+
+The locale element can be used to extract the data for a specific locale. For example, if a bundle defines some metatype description in French:
+
+```
+    <configuration>
+        <reports>
+            <report>
+                <outputFile>metadata.xml</outputFile>
+                <locale>fr-FR</locale>
+            </report>
+        </reports>
+    </configuration>
+```
+
+#### Customization of the Report Content
+
+In some case, it may be necessary to control what data should be present in the report, for example, if you want to extract further data from a project with plugins. For this you can use the `reportConfigs` element to create named configuration of reports. Then, with the `configName` element you can reference the configuration name that will be used to extract and aggregate the data of the report.
+
+See the documentation [here](https://bnd.bndtools.org/instructions/reportconfig.html) for more information.
+
+##### Example
+
+For example, the following configuration will generate the file `metadata.json` with the specified `myConfigName` configuration name. This configuration will not use any default extraction plugins (`clearDefaults`), it will import the `configuration.json` file of the bundle into the report and we also add an arbitrary variable: 
+
+```
+    <configuration>
+        <reportConfigs>
+            <myConfigName>
+                <clearDefaults>true</clearDefaults>
+                <reportPlugins>
+                    <reportPlugin>
+                        <pluginName>importJarFile</pluginName>
+                        <properties>
+                            <path>OSGI-INF/configuration.json</path>
+                        </properties>
+                    </reportPlugin>
+                </reportPlugins>
+                <variables>
+                    <myVariable>myValue</myVariable>
+                </variables>
+            </myConfigName>
+        </reportConfigs>
+        <reports>
+            <report>
+                <outputFile>metadata.json</outputFile>
+                <configName>myConfigName</configName>
+            </report>
+        </reports>
+    </configuration>
+```
+
+The resulting `metadata.json` file could be:
+
+```json
+{
+  "configuration":{<the configuration from the configuration.json file>}
```

Comment:
I think your json is problematic. The `<>` are flagged and there is no comma between the map entries.

Reply:
This was a comment, I placed it with sample values.